src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -751,10 +757,11 @@ public struct OperationBlockAnalysisContext
         /// </summary>
         public CancellationToken CancellationToken => _cancellationToken;
 
-        public OperationBlockAnalysisContext(ImmutableArray<IOperation> operationBlocks, ISymbol owningSymbol, AnalyzerOptions options, Action<Diagnostic> reportDiagnostic, Func<Diagnostic, bool> isSupportedDiagnostic, CancellationToken cancellationToken)
+        public OperationBlockAnalysisContext(ImmutableArray<IOperation> operationBlocks, ISymbol owningSymbol, SemanticModel semanticModel, AnalyzerOptions options, Action<Diagnostic> reportDiagnostic, Func<Diagnostic, bool> isSupportedDiagnostic, CancellationToken cancellationToken)
```

Comment:
All the context type constructors were made public prior to 1.0.0 release to enable 3rd party analyzer authors to write their own test frameworks to run analyzers. Apart from that, there should be no public consumers that want to instantiate context types as the analyzer driver does it for them.